client/actionView/action.html

```diff
@@ -1,16 +1,13 @@
 
-<link rel="stylesheet" type="text/css" href="action.css">
+<link rel="stylesheet" type="text/css" href="actionView/action.css">
 <!-- IMPORTANT -->
 <!-- Added for MODULAR testing only. Remove when done. -->
-<script src='https://cdnjs.cloudflare.com/ajax/libs/angular.js/1.4.5/angular.js'></script>
 <div class="actionWrapper">
 
 
 <script type="text/javascript" src="action.js"></script>
```

Comment:
Action.js is the actionController.js file now, and has been moved to the Index.file. Sola can rearrange his file to include a actionServices.js file.